.werft/values.dev.yaml

```diff
@@ -103,6 +103,7 @@ components:
     serviceType: "ClusterIP"
     deployIngressService: false
     loadBalancerIP: null
+    sslDHParam: LS0tLS1CRUdJTiBESCBQQVJBTUVURVJTLS0tLS0KTUlJQkNBS0NBUUVBdkhsWGFIOHlyNWFJYWhmc3N5RDltTDJXL2NQZUdhWmtwdUF6dm1KamFXeEZqUFJrM2IvRworbDNJY3pTRndVVy9YN2NjU2NlZ1JzZjl1dFJKR1hWZWMyL0c1VFlETkJsNDdMQ1I1VXMvUmpsV0tHcWJTVnVhCnYxY3BsUUgvVDl4UGJ4bjE2QjhrOFZmdEVUNEgxTmhRM2lxSWtGbUk2QTZrTDJzbi9VQjdhMXYydDRjL3VpdVcKcGtNNlVaRlVoUGhhVkt6VWJsR0pTWEdnbjdiVUtGWDNQSVdLY1VHM015NmhYVWg0TzRLTXhaYWRWS051T011SApyK0JLZDdsWkx5SmlBQWRtWUtTaEpNVGUwSS9uRTlFR3Q0VHVGZVZNaXpkbHR5SWkyWEFLRUlwYjZuTFlHWjY1CkNPRkx2WFpXOTJOK0U0ZFdES1ozaHBTbk1qclptVEIvYXdJQkFnPT0KLS0tLS1FTkQgREggUEFSQU1FVEVSUy0tLS0tCg==
```

Comment:
I'm not sure I understand the role of this parameter properly, but wonder if couldn't fold this into the certificate secret? AFAIK we even have a `params` file there.

Reply:
@csweichel a Diffie-Hellman parameter is a set of randomly generated data used when establishing Perfect Forward Secrecy during initiation of an HTTPS connection. Is not related to the SSL certificate.
Without this setting, DHE ciphers cannot be used. http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_dhparam

This post provides more details https://raymii.org/s/tutorials/Strong_SSL_Security_On_nginx.html#toc_5

Reply:
>  Is not related to the SSL certificate.

Absoluteley, makes sense.